src/ui/components/filters/facetscomponent.js

```diff
@@ -272,23 +273,21 @@ export default class FacetsComponent extends Component {
   }
 
   /**
-   * Applies default formatting to different types of facets
+   * Applies default formatting to a facet
    *
-   * @param {DisplayableFacet[]} facets from answers-core
-   * @returns {DisplayableFacet[]} from answers-core
+   * @param {DisplayableFacet} facet from answers-core
+   * @returns {DisplayableFacet} from answers-core
    */
-  _processFacets (facets) {
+  _processFacet (facet) {
```

Comment:
I think we may want to give this method a less generic name. It's a bit hard to differentiate between `transformFacets` and `processFacet`. We want a name that also indicates why this should be run after `transformFacets`. What about `applyDefaultFormatting` or something like that?

Reply:
ya, I like that